doc/stages/filters.voxeldownsize.rst

```diff
@@ -39,9 +40,11 @@ cell
 
 mode
   Mode for voxel based filtering. [Default: voxelcenter]
-  **voxelcenter**: Retain center of a populated voxel. Coordinates of a first point detected will be modified and set to the center of a populated voxel.
-  **firstinvoxel**: Retain first point detected in each voxel.
+  **voxelcenter**: Coordinates of the first point found in each voxel will
+  be modified to be the center of the voxel.
+  **firstinvoxel**: Only the first point found in each voxel is retained.
```

Comment:
Docs show these as `voxelcenter` or `firstinvoxel` with the former being the default. The updated code indicates the choices would be `center` or `first` with `center` still being the default. A call to PDAL options however displays the following message, which is misleading.

```
$ pdal --options=filters.voxeldownsize
filters.voxeldownsize -- http://pdal.io/stages/filters.voxeldownsize.html
-----------------------------------------------------------------------------------------------------
  user_data
      User JSON

  log
      Debug output filename

  option_file
      File from which to read additional options

  cell [0.001]
      Cell size

  mode [centerfirst]
      Method for downsizing : center / first
```

Reply:
Oops